docs/md/modules-external-exome-reprocessing.md

```diff
@@ -272,16 +243,44 @@ Queries the WFL database for workloads. Specify the project name to query for a
     }]
     ```
 
-The "workflows" field lists out each Cromwell workflow that was started, and
-includes their status information. It is also possible to use the Job Manager
-to check workflow progress and easily see information about any workflow
-failures.
-
 !!! warning "Note"
```

Comment:
This is no longer true, these are query parameters now.

Reply:
They always were...